src/VerifyWebauthnController.js

```diff
@@ -139,10 +139,10 @@ export default FormController.extend({
   Form: {
     autoSave: true,
     hasSavingState: false,
-    title: _.partial(loc, 'factor.webauthn.biometric', 'login'),
```

Comment:
we shouldn't change this. If I remember correctly we needed `_partial` because we had some problem loading a new language since the language was not loaded in time when this was calculated. Adding this as a function was the workaround

Reply:
nice catch.

Reply:
@haishengwu-okta  how do we avoid this issue in idx ? Also I wonder if this issue can be solved in some other way.

Reply:
we need to use same approach in IDX.